src/Worker/Grpc/GrpcOrchestrationRunner.cs

```diff
@@ -223,7 +207,7 @@ public static string LoadAndRun(
 
                 if (addToExtendedSessions && !executor.IsCompleted)
                 {
-                    extendedSessions.Set<ExtendedSessionState>(
+                    extendedSessions!.Set<ExtendedSessionState>(

```

Comment:
Same feedback about adding a comment. This is probably a good thing to always do when using the `!` operator, because it's rarely something we should be using and could therefore use an explanation.